### PR TITLE
CE-3695: Fix spacing

### PR DESCRIPTION
**Pull request by lukaszkonieczny**

### CODE REVIEW

**Comment by kamilkoterba** [comment]
LGTM :)

**Comment by kamilkoterba** [comment]
LGTM :)

File: extensions/wikia/PotentialMemberPageExperiments/styles/entry-point-experiment.scss [hunk 1]
```diff
@@ -15,17 +16,18 @@ $close-button-background: '/resources/wikia/ui_components/modal/images/close-dar
 	background: url('/extensions/wikia/PotentialMemberPageExperiments/images/858-px-background.png') no-repeat center bottom;
 	background-size: cover;
 	border-bottom: 2px solid $color-blue;
-	height: 150px;
```
Comment (lukaszkonieczny) [inline extensions/wikia/PotentialMemberPageExperiments/styles/entry-point-experiment.scss:19]: I'm not sure is it safe to remove this property. Right now banner in article is smaller and has cut off background

File: extensions/wikia/PotentialMemberPageExperiments/styles/entry-point-experiment.scss [hunk 2]
```diff
@@ -15,17 +16,18 @@ $close-button-background: '/resources/wikia/ui_components/modal/images/close-dar
...
 	margin: 18px 0;
 	overflow: hidden;
+	padding-bottom: $banner-bottom-space;
```
Comment (lukaszkonieczny) [inline extensions/wikia/PotentialMemberPageExperiments/styles/entry-point-experiment.scss:26]: Glue banner in narrow screen is too high now

Comment (kamilkoterba) [inline extensions/wikia/PotentialMemberPageExperiments/styles/entry-point-experiment.scss:26]: I added this for glue banner, so should be same size as previously `height: calc(320px - #{$banner-bottom-space});`

File: extensions/wikia/PotentialMemberPageExperiments/styles/entry-point-experiment.scss (hunk 1 above)
Comment (kamilkoterba) [inline extensions/wikia/PotentialMemberPageExperiments/styles/entry-point-experiment.scss:19]: added min height

**Comment by lukaszkonieczny** [comment]
LGTM

**Pull request merged**